languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -139725,19 +139734,6 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <suggestion>faire des coupes claires</suggestion>
             <example correction="faire des coupes claires"><marker>faire des coupes sombres</marker></example>
         </rule>
-        <rule id="TOMBER_DANS_LES_POMMES" name="tomber dans les pommes">
```

Comment:
Moved to style (it's interesting to know, but I don't think it should stay in grammar)

Reply:
though: maybe we should not even keep it at all. The sources seems confused on the origin or even correctness of the whole thing 🤔 
But at least yeah, that's not grammar at all, good catch!
![image](https://github.com/languagetool-org/languagetool/assets/114988314/53b774a5-e712-41df-a165-a9a0c87399c2)